overlord/devicestate/devicestate_remodel_test.go

```diff
@@ -5194,7 +5199,7 @@ func (s *deviceMgrRemodelSuite) TestRemodelTasksSelfContainedModelMissingDepsOfM
 		c.Check(fromChange, Equals, "99")
 		c.Check(opts.Channel, Equals, "latest/stable")
 
-		return nil, nil
+		return state.NewTaskSet(), nil
```

Comment:
It isn't realistic (when comparing it to the actual impl) for this mocked function to return a nil task set and a nil error.